### PR TITLE
MM-47301: Migrate /enterprise/system_console/about e2e tests to TypeScript 

### PR DESCRIPTION
**Pull request by adithyaakrishna**
#### Summary

This PR Migrate /enterprise/system_console/about e2e tests to TypeScript

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/21330
Fixes https://mattermost.atlassian.net/browse/MM-47301

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Comment by adithyaakrishna** [comment]
Hey @tboulis 
Okay sure, I'll address the changes and send a commit regarding that :)

File: e2e/cypress/tests/integration/enterprise/system_console/about/edition_and_license_spec.ts [hunk 1]
```diff
@@ -130,7 +132,8 @@ function verifyRenamePrivateChannel(teamName, privateChannelName, testUsers) {
     }
 }
 
-function verifyUserChannelPermission(teamName, privateChannelName, sysadmin, teamAdmin, regularUser) {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function verifyUserChannelPermission(teamName: string, privateChannelName: string, sysadmin: { username: string; password: string; email: string }, teamAdmin: any, regularUser: any) {
```
[the remark this answers is not in the document]
Comment (adithyaakrishna) [reply e2e/cypress/tests/integration/enterprise/system_console/about/edition_and_license_spec.ts:136]: Gotcha and Done 💯

**Comment by adithyaakrishna** [comment]
@tboulis I have a small doubt, this line of code is giving an error when Cypress.Team is used. Could you please help me with this?

Screenshot 2022-10-20 at 11 38 15 PM

**Comment by lieut-data** [comment]
Heads up that as part of our efforts to move to a monorepo, we're closing out a number of older pull requests like this one to help streamline the effort.

If you'd like to preserve these changes -- even if you're not the original author! -- feel free to resubmit the pull request against the monorepo once it's ready. You can subscribe to [mattermost-server-issue-22420](https://www.github.com/mattermost/mattermost-server/issues/22420) for status updates on this effort.

**Pull request closed without merging**